src/index.html

```diff
@@ -11,7 +11,182 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+    >
+      <fieldset class="fieldset">
+        <legend>Personal information</legend>
+        <div class="field-form">
+          <label for="surname">
+            Surname:
+            <input type="text"
+            autocomplete="off"
+            name="surname"
+            id="surname"
+            >
+          </label>
+        </div>
```

Comment:
remove these redundant divs, you can apply `field-form` class to label directly, also [checklist](https://github.com/mate-academy/layout_html-form/blob/master/checklist.md) 3rd item, fix your attributes formatting everywhere
```suggestion
          <label clase="field-form" for="surname">
            Surname:
            <input 
              type="text"
              autocomplete="off"
              name="surname"
              id="surname"
            >
          </label>
```